lib/fs-driver.js

```diff
@@ -30,7 +30,9 @@ class FsDriver {
     if (!isString(str)) {
       str = str.toString()
     }
-    return str.replace(/(['/\\])/g, '\\$1')
+    return str.replace(/'/g, '\'"\'"\'')
```

Comment:
anyway, @rsandor can you put some comment what you are doing here and why. Would be helpful for the future